bundle/manifests/apicast-operator.clusterserviceversion.yaml

```diff
@@ -96,12 +96,7 @@ spec:
                 app: apicast
                 com.company: Red_Hat
                 control-plane: controller-manager
-                rht.comp: 3scale_apicast
                 rht.comp_ver: "2.14"
-                rht.prod_name: Red_Hat_Integration
```

Comment:
hmm
trying to understand something, I thought we have agreed to leave these as they are and undo change to the failing test? I can see you have removed change to tests but why is it passing now?

Reply:
Yes, I did undo in test file, and now it has no changes - this is place that was commented before. - https://github.com/valerymo/apicast-operator/blob/apicast-community-v0.8.1/test/manifests-version/deployment_version_test.go#L58C1-L59

It required  `rht.comp.ver`:
```
if deployment.Spec.Template.Labels["rht.comp_ver"] != version.ThreescaleRelease {
```
So I have just this `rht.comp_ver: "2.14"` in CSV and test passed.
Thank you

Reply:
Do you want restore all rht in CSV?

Reply:
yes, I think we can have all of them restored, they are not doing any harm but causing unnecessary work in both, upstream and midstream. Let's restore.

Reply:
Done. Thank you for comments @MStokluska